selenium_tests/pages.py

```diff
@@ -0,0 +1,78 @@
+from typing import List
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webdriver import WebDriver
+
+
+class BasePage():
+
+    def __init__(self, driver: WebDriver) -> None:
+        self.driver = driver
+
+    def url_matches(self, path: str) -> bool:
+        return self.driver.current_url.endswith(path)
```

Comment:
can we use [URL parsing](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse)?

Reply:
Good idea - this has been added.